llama/runner/runner.go

```diff
@@ -93,6 +93,17 @@ type NewSequenceParams struct {
 	embedding      bool
 }
 
+type multiLPath []string
+
+func (m *multiLPath) Set(value string) error {
+	*m = append(*m, value)
+	return nil
+}
+
+func (m *multiLPath) String() string {
+	return strings.Join(*m, ", ")
+}
+
```

Comment:
Can you move multiLPath and its receivers down so that it is closer to where it is used - i.e. above loadModel?